apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
+    text-align: center;
+  }
+  
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
```

Comment:
Ideally these colors should be defined in some theme.scss file and then should be used as variables here.
```
For example:-
$primary_color: #61dafb;
```
and then using it here as 
```
color: $primary_color
```

Also this means switching to a pre-processor like `scss` or `less` or css in js.
Whatever we chose but one of them is needed.

Reply:
Ohh okay, thanks a lot for the example as well. Will make the changes.